src/components/new-project/index.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+import { NewProject, NewProjectProps } from './'
+import icon from '../../images/bulb.png'
+
+const story: Meta = {
+  title: 'NewProject/Default',
```

Comment:
Skvělé, že komponenta byla přidána do Storybooku. 

Nemáš náhodou nápad na obecné pojmenování? Také navrhuji zanořit do modulu `cards`. 

V případě struktury Storybooku navrhuji upravit na následující tvar.

```typescript
const story: Meta = {
  title: 'Cards/NewProject', // Možná to obecné pojmenování
  component: NewProject,
}

export const Default = Template.bind({})
```

Reply:
No, už jsem nad tím přemýšlel při implementaci, že je to hodně podobné těm cards komponentám (https://github.com/cesko-digital/web/issues/69). Možná bych to sjednotil, co myslíš? Tím by se pak mohlo asi vyřešit obecné pojmenování zároveň

Reply:
Zatím bych nechal takto a obecné řešení začal řešit, až nám to začne růst.

Reply:
Hotovo.